### ISSUE TITLE
Spectrum scaling compressed in comparison to sdrangel and gqrx

### ISSUE THREAD

**Issue by jbvoelker** [demantz/RFAnalyzer#68]
I am currently using RFAnalyzer on a tablet connected to a RTL-SDR as part of my Es'Hail-2 receiving setup.
This works great, but the signal levels on the spectrum view seem to be compressed compared to the spectrum display in gqrx and sdrangel.
For example, the beacon signal which peaks ~ 25 dB above the transponder noise with sdrangel and gqrx peaks ~ 15 dB above the transponder noise in RFAnalyzer.

I have built an apk with line 217 in AnalyzerProcessingLoop.java changed to
`mag[targetIndex] = (float) (10* Math.log10(realPower + imagPower));`
and the resulting spectrum display now seems to be consistent with gqrx and sdrangel.

For comparison, [this](https://github.com/csete/gqrx/blob/a8e98683aa05c5e554eb689994f87879a7abcfe3/src/applications/gqrx/mainwindow.cpp#L1284) is how the scaling is handled in gqrx.

Can somebody confirm the issue, or is it just a problem with my RFAnalyzer settings?

**Comment by wissanu020430** [comment]
https://github.com/demantz/RFAnalyzer/issues/69#issuecomment-499084698

**Comment by demantz** [comment]
You are absolutely correct. Thank you very much for reporting this. It is fixed in the new RF Analyzer 2.0 version which releases soon..
